### DISCUSSION TITLE
✅ 🔗 Editor Top Bar - Option to View Page (Duplicate of #21925) - [ED-14720]

### DISCUSSION THREAD

**Discussion by mmg68** [elementor/elementor#24491] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

After updating to the new editor, there is only an option to exit to WordPress, when there was an option to view page on exiting the editor.

### Describe the solution you'd like

Add back the View Page option

### Describe alternatives you've considered

If this change was made to streamline the UI, then just provide the option to View Page upon exiting the editor.

**Comment by nicholaszein** [comment]
Hello there, @mmg68!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21925

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hi @mmg68!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
